jans-config-api/plugins/admin-ui-plugin/src/test/java/io/jans/ca/plugin/adminui/service/webhook/WebhookServiceTest.java

```diff
@@ -0,0 +1,102 @@
+package io.jans.ca.plugin.adminui.service.webhook;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.slf4j.Logger;
+
+import io.jans.ca.plugin.adminui.model.exception.ApplicationException;
+import io.jans.ca.plugin.adminui.model.webhook.WebhookEntry;
+import io.jans.ca.plugin.adminui.utils.ErrorResponse;
+import io.jans.configapi.configuration.ConfigurationFactory;
+import io.jans.configapi.model.configuration.ApiAppConfiguration;
+
+class WebhookServiceTest {
+
+    @InjectMocks
+    private WebhookService webhookService;
+
+    @Mock
+    private ConfigurationFactory configurationFactory;
+
+    @Mock
+    private ApiAppConfiguration apiAppConfiguration;
+
+    @Mock
+    private Logger log;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    void testValidateWebhookEntry_BlockedUrl() {
```

Comment:
In settings we have blocked url "192.168.". Add positive and negative cases for this particular block. We can't list all urls as in tests below. We need "pattern". Something like this https://github.com/JanssenProject/jans/blob/6c1caa1c4c92d28571f8589cd701e6885d4d85ef/jans-auth-server/client/src/test/java/io/jans/as/client/ws/rs/ClientWhiteListBlackListRedirectUris.java#L41